### PR TITLE
Updated Readme and Profile template

### PR DESCRIPTION
**Pull request by JenBauer**
- Added more detail to README.md and adjusted a few spots for style/grammar
- Added a markdown template doc for possible use in future Issues as a simple update project for chapter members or folks to use as their own profile readme when getting started with a new account

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,54 +25,55 @@
...
-- 🤓 You must make 4 PR (pull requests) and have 4 MR (merge requests)
-- 🌲 You will get a tree planted in your name and some cool virtual swag
-- 🎃 Join the slack chanel #hacktober-2023 and ping a lead for help 
-- 👀 Check the WWCode Greenville Notion doc if you need help with GitHub or contributing
+<!-- Hacktoberfest 2023 -->
+## 🎃 Hacktoberfest 2023
+- 🍴 To get started, fork this repository.
+- 👯‍♀️ Clone the repo to your computer.
+- 👩🏽‍💻 Participate by closing our open issues and contributing to the docs.
+- 🤓 You must make 4 PR (pull requests) and have 4 MR (merge requests) to complete the challenge!
```
Comment (klmansel) [inline README.md:50]: @JenBauer can you update this to 
```suggestion
- 🤓 You must have 4 PRs (pull requests) merged to complete the challenge!
```

**Review by klmansel — APPROVED** [review]
One request, then approved!

File: README.md (hunk 1 above)
Comment (klmansel) [reply README.md:50]: The actual wording is:

>  _YOUR PR/MRS MUST BE MERGED, HAVE THE “HACKTOBERFEST-ACCEPTED” LABEL, OR HAVE AN OVERALL APPROVING REVIEW._

so I'd be fine w/adjusting this too

**Comment by JenBauer** [comment]
Also, this PR needs the event label added - should we (submitters) add the label beforehand, or does the approver add the "hacktoberfest-approved" label _after_ approving?

**Comment by klmansel** [comment]
> Also, this PR needs the event label added - should we (submitters) add the label beforehand, or does the approver add the "hacktoberfest-approved" label _after_ approving?

Either, as long as it's before it's merged! The issues should have the label in the first place though.

**Comment by JenBauer** [comment]
@klmansel This should be good to go for you.  I can make more issues based on the template from this one.

**Pull request merged**